app/models/service.rb

```diff
@@ -109,7 +109,11 @@ def custom_actions
   end
 
   def custom_action_buttons
-    service_template&.custom_action_buttons(self)
+    if service_template.nil?
+      CustomButton.where(:applies_to_class => "Service")
+    else
+      service_template.custom_action_buttons(self)
+    end
```

Comment:
This looks fine, although I might suggest `if service_template` and swapping the if-else blocks.  Doing a 1-line ternary operator comes out to 121 characters.  So close.

Anyway, the real reason I'm here is to ask about the `custom_actions` method just above this method which looks like it will suffer the same fate and should be addressed as well.